packages/Textarea/src/Textarea.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import cx from 'classnames';
+import autosize from 'autosize';
+
+import './textarea.scss';
+
+export interface TextareaProps extends React.ComponentProps<'textarea'> {
+  /** True if the textarea should allow new lines with Enter */
+  allowNewline?: boolean;
+  /** Add class names to the surrounding DOM container. */
+  className?: string;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
+  /** True if the textarea has an error. */
+  error?: boolean;
+  /** True if the textarea should resize
+   * automatically when a multiline value is set. */
+  isAutoResize?: boolean;
+  /** The maximum number of characters allowed. Required if
+   *  "showCharactersIndicator" is enabled. */
+  maxLength?: number;
+  /** Function called when the value changes. */
+  onChange?: (e: React.ChangeEvent<HTMLTextAreaElement>) => void;
+  /** True to display the number of remaining allowed characters
+   *  in the bottom right of the text area. Requires a valid
+   *  value in the "maxLength" prop. */
+  showCharactersIndicator?: boolean;
+  /** Specifies the value inside the input. */
+  value?: string;
+}
+
+const Textarea: React.FunctionComponent<TextareaProps> = (
+  props: TextareaProps
+) => {
+  const {
+    allowNewline = true,
+    className,
+    dataTest,
+    error = false,
+    isAutoResize = false,
+    maxLength,
+    onChange,
+    showCharactersIndicator = false,
+    value,
+    ...rest
+  } = props;
+  const textareaRef = React.useRef<HTMLTextAreaElement | null>(null);
+  const [currentCharLength, setCurrentCharLength] = React.useState(
+    value?.length ?? 0
+  );
+  const [currentValue, setCurrentValue] = React.useState(value ?? '');
+  const textareaMaxLength = maxLength ?? 0;
+  const displayCharIndicator = showCharactersIndicator && textareaMaxLength > 0;
+
+  const classes = cx('ids-textarea', className, {
+    'ids-textarea--has-char-count': displayCharIndicator,
+  });
+
+  const fieldClasses = cx('ids-textarea__field', {
+    'ids-textarea__field--error': error,
+    'ids-textarea__field--disabled': rest.disabled,
```

Comment:
Most of the time `disabled` is a specific component property.

We should see it in the `TextareaProps` interface also.

Reply:
Fixed